src/apiCalls.js

```diff
@@ -1,7 +1,15 @@
-// Your fetch requests will live here!
-// Write fetch call
-// Export function to scripts.js
-// Import function in scripts.js
-// Invoke function in scripts.js
+// Fetch requests
 
-console.log('I will be a fetch request!')
\ No newline at end of file
+const usersApi = 
+fetch("https://what-s-cookin-starter-kit.herokuapp.com/api/v1/users")
+  .then(response => response.json())
+
+const recipesApi = 
+fetch("https://what-s-cookin-starter-kit.herokuapp.com/api/v1/recipes")
+  .then(response => response.json())
+
```

Comment:
We're just parsing the data in this file and then we're exporitng the promise to the scripts file